linux_os/guide/services/mail/package_s-nail_installed/policy/stig/shared.yml

```diff
@@ -21,3 +21,6 @@ checktext: |-
 
 vuln_discussion: |-
     The "s-nail" package provides the mail command required to allow sending email notifications of unauthorized configuration changes to designated personnel.
+
+vuldiscussion: |-
```

Comment:
I'm confused why there is `vuln_discussion` vs `vuldiscussion`, I think we should unify them into a single field or is there a reason for that?

Reply:
There are many occurrences of both in the project

Reply:
Yes. I agree we should unify them. But I believe this could also be solved in a separate PR, to make the review easier.